src/components/SingleProduct.js

```diff
@@ -65,6 +72,13 @@ const SingleProduct = ({products, setProducts}) => {
         </div> : null
     
         }    
+        <div className="sealContainer">
```

Comment:
Instead of a div, consider a footer as the div may leave a lot of empty space at the bottom of the page.